encoding/protobuf.go

```diff
@@ -71,11 +71,11 @@ func (p protoSerializer) Response(body *transport.Response) (interface{}, error)
 	if err != nil {
 		return nil, err
 	}
-	var objmap map[string]*json.RawMessage
-	if err = json.Unmarshal(str, &objmap); err != nil {
+	var unmarshaledJson json.RawMessage
```

Comment:
nittiest of nits: per Go naming conventions, this should be unmarshaledJSON :)

https://github.com/golang/go/wiki/CodeReviewComments#initialisms